### PR TITLE
Feature/travisfile

### PR DESCRIPTION
**Pull request by himaninare**

### CODE REVIEW

File: src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java [hunk 1]
```diff
@@ -4,21 +4,26 @@
...
 import com.fasterxml.jackson.core.JsonProcessingException;
 import io.restassured.response.Response;
+import org.apache.log4j.Logger;
 import org.testng.annotations.Test;
 
 import static com.ccs.conclave.api.cii.verification.VerifyResponses.*;
 import static com.ccs.conclave.api.common.SchemeRegistry.*;
 
-public class GetSchemeInfoTests {
+public class GetSchemeInfoTests extends BaseClass {
```
Comment (amrutha-sireesh) [inline src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java:16]: Do we need to extend BaseClass here?

File: src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java [hunk 2]
```diff
@@ -4,21 +4,26 @@
 import com.ccs.conclave.api.cii.pojo.SchemeInfo;
 import com.ccs.conclave.api.cii.requests.RestRequests;
 import com.ccs.conclave.api.cii.response.SchemeInfoResponse;
+import com.ccs.conclave.api.common.BaseClass;
 import com.fasterxml.jackson.core.JsonProcessingException;
```
Comment (amrutha-sireesh) [inline src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java:8]: can you please remove unused files as well?

Comment (himaninare) [reply src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java:8]: Its needed as we are extending Baseclass here

File: src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java (hunk 1 above)
Comment (himaninare) [reply src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java:16]: Yes

**Review by amrutha-sireesh — APPROVED** [review]

File: src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java [hunk 3]
```diff
@@ -4,21 +4,26 @@
...
+
+    private final static Logger logger = Logger.getLogger(GetSchemeInfoTests.class);
 
     @Test
     public void getCompaniesHouseSchemeInfo() {
         OrgDataProvider orgData = new OrgDataProvider();
         SchemeInfo schemeInfo = orgData.getInfo(COMPANIES_HOUSE);
         Response response = RestRequests.getSchemeInfo(COMPANIES_HOUSE, schemeInfo.getIdentifier().getId());
         verifyGetSchemeInfoResponse(orgData, response.as(SchemeInfoResponse.class));
+        logger.info("Response is"+ response.asString());
```
[the remark this answers is not in the document]
Comment (himaninare) [reply src/test/java/com/ccs/conclave/api/cii/tests/GetSchemeInfoTests.java:26]: Updated the log4j properties

**Pull request merged**